docs/README.md

```diff
@@ -11,24 +11,24 @@
 - [Dumping](#dumping)
 
 ## Core concepts
-**clin** sends http requests to Nakadi to create or update resources. The source
-of data for this is yaml manifest file which corresponds to a single Nakadi
+**clin** sends HTTP requests to Nakadi to create or update resources. The source
+of data for this is YAML manifest file which corresponds to a single Nakadi
 resource.
 
 These manifests can be applied:
 - directly one by one, see
-  [Applying single manifest](#applying-single-manifest).
+  [Applying a single manifest](#applying-single-manifest).
 - in batch by using a "clin file" to roll out whole infrastructure with a single
   command, see [Batch processing](#batch-processing).
 
 `{{TEMPLATE_VARIABLES}}` and `@@@./includes.yaml` , optionally enforced by
-[yaml anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
+[YAML anchors](https://www.google.com/search?q=yaml+anchors) enable expressive
 and DRY definition of the Nakadi infrastructure.
 
 For more details see [Manifests format](#manifests-format).
 
 ### Dry-run by default
-Each command that can possibly change Nakadi resources in NOT performing any
+Each command that may change Nakadi resources is NOT performing any
```

Comment:
I would rephrase this to something like:

Commands do NOT execute any changes (write operations?) by default.

Reply:
Noted, will rephrase. Thanks.